refresh-collaborators.py

```diff
@@ -0,0 +1,44 @@
+import os
```

Comment:
We need to have the Apache license at the top of the file.

Reply:
Wil do.